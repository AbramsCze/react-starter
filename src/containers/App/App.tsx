// libs
import React, { Fragment, FunctionComponent } from 'react'
import { Route, Switch } from 'react-router'
//import { ToastContainer } from 'react-toastify-redux'

// others
//import { Toast } from '../../components'
import routes from '../../constants/routes'
import { AddUserPage } from '../Users/AddUserPage'
import { Homepage } from '../Homepage/Homepage'

export const App: FunctionComponent = () => (
  // <ToastContainer toastComponent={Toast} position="top-right" />
    <Fragment>
      <Switch>
        <Route path={routes.USER_FORM} component={AddUserPage} />
        <Route path={routes.EMPTY} component={Homepage} />
      </Switch>
    </Fragment>
)

export default App
