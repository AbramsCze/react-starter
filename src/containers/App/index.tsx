import React, { Component, Fragment } from 'react'
import { RouteComponentProps } from 'react-router'
import { LoadingButton } from '../../components'

export class App extends Component<RouteComponentProps<void>> {
  render() {
    return (
      <Fragment>
        <LoadingButton />
      </Fragment>
    )
  }
}
