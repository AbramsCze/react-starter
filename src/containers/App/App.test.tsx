// libs
import React from 'react'
import { shallow } from 'enzyme'

// others
import { App } from './App'

describe('App page', () => {
  it('should render', () => {
    const renderedApp = shallow(<App />)
    expect(renderedApp).toMatchSnapshot()
  })
})
