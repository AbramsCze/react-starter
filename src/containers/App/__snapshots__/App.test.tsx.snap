// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App page should render 1`] = `ShallowWrapper {}`;
